src/components/table/row/Row.tsx

```diff
@@ -0,0 +1,47 @@
+/// <reference types="vite-plugin-svgr/client" />
+import MoreInfoDots from "@/assets/moreInfo_dots.svg?react";
+import InformationCircle from "@/assets/information-circle.svg?react";
+import Typography from "@components/typography/Typography";
+import React, { ReactNode, useState } from "react";
+import Menu from "@components/menu/Menu";
+import { MenuItem } from "@components/menu/items";
+
+type RowProps = {
+  children: ReactNode;
+  menuItems?: MenuItem[];
+};
+
+const Row = React.memo(({ children, menuItems = [] }: RowProps) => {
+  const [isMenuOpen, setIsMenuOpen] = useState<boolean>(false);
+  const [target, setTarget] = useState<HTMLDivElement | null>(null);
```

Comment:
why not use useRef instead of useState?

Reply:
[useRef](https://react.dev/reference/react/useRef) wouldn't trigger re-renders, which leads to inconsistencies in the UI.